test/config.yaml

```diff
@@ -13,13 +13,11 @@ Logger:
 LegacyMetrics:
   Enabled: true
   Dataset: refinery_metrics
-  APIKey: $REFINERY_HONEYCOMB_API_KEY
```

Comment:
Why are these deleted?

Reply:
Because the $ syntax doesn't work, but there are environment variables that are automatic if they're set.